src/btree/bt_debug.c

```diff
@@ -699,16 +701,38 @@ static int
 __debug_dsk_col_fix(WT_DBG *ds, const WT_PAGE_HEADER *dsk)
 {
     WT_BTREE *btree;
+    WT_CELL_UNPACK_KV unpack;
+    WT_COL_FIX_AUXILIARY_HEADER auxhdr;
     uint32_t i;
     uint8_t v;
 
     btree = S2BT(ds->session);
 
-    WT_FIX_FOREACH (btree, dsk, v, i) {
+    WT_RET(__wt_col_fix_read_auxheader(ds->session, dsk, &auxhdr));
+    switch (auxhdr.version) {
+    case WT_COL_FIX_VERSION_NIL:
+        WT_RET(ds->f(ds, "page version 0, no auxiliary data\n"));
+        break;
+    case WT_COL_FIX_VERSION_TS:
+        WT_RET(ds->f(ds, "page version 1, %" PRIu32 " time windows\n", auxhdr.entries));
+        break;
+    default:
+        WT_RET(ds->f(ds, "unknown page version %" PRIu32 "\n", auxhdr.version));
```

Comment:
For debug it seems better to just print it. If you got to the point where you're calling debug on a page with a bad version, probably you don't want the error :-) (btw: I changed this code around some, because of the changes Keith wanted in __wt_col_fix_read_auxheader, but it still works the same way.)

Reply:
ok.